docs/source/conf.py

```diff
@@ -56,9 +56,9 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'OSS AtlasDB'
-copyright = u'2017, Palantir Technologies'
-author = u'Palantir Technologies'
+project = 'OSS AtlasDB'
+copyright = '2017, Palantir Technologies'
+author = 'Palantir Technologies'
 
 def get_last_release_from_gradle():
```

Comment:
`subprocess.Popen.communicate` is changed in Python 3 to return byte strings, so we'll need to decode the output before applying the regex matcher against it.

Reply:
The `stdout = p.communicate()[0]` in this function needs to be something like `stdout = p.communicate()[0].decode("utf-8")` (maybe check if the encoding is necessarily utf-8 first)

Reply:
💃 thanks!